src/redprl/refiner.fun

```diff
@@ -1477,25 +1477,27 @@ struct
          | (Syn.FST _, Syn.FST _, _) => DProd.FstEq
          | (Syn.SND _, Syn.SND _, _) => DProd.SndEq
          | (Syn.PATH_AP (_, P.VAR _), Syn.PATH_AP (_, P.VAR _), _) => Path.ApEq
-         | (Syn.PATH_AP (_, P.APP _), _, _) => Path.ApComputeConst
+         | (Syn.PATH_AP (_, P.APP _), _, _) => Path.ApConstCompute
          | (_, Syn.PATH_AP (_, P.APP _), _) => Equality.Symmetry
          | _ => raise E.error [E.% "Could not find neutral equality rule for", E.! m, E.% "and", E.! n, E.% "at type", E.! ty]
 
-      fun StepEqEta ty =
-        case Syn.out ty of
-           Syn.DPROD _ => DProd.Eta
-         | Syn.DFUN _ => DFun.Eta
-         | Syn.PATH_TY _ => Path.Eta
-         | _ => raise E.error [E.% "Could not find eta expansion rule for type", E.! ty]
+      fun StepEqNeuVal (m, ty) =
```

Comment:
@favonia Nah, I was thinking of new names for `StepEqNeuVal`, `StepEqValVal`. Anyway, the code as-written looks correct. Reserving `Compute` for beta-like rules quite sensible, so maybe we can think of something else for other principles, as you say.

In any case, if you don't want to think about names just yet, feel free to merge whenever you like, since this looks like a good change.